### PR TITLE
dnstracer: add page

### PR DESCRIPTION
**Pull request by Faro1991**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**
1.9, ref: https://www.kali.org/tools/dnstracer/

### CODE REVIEW

**Comment by Faro1991** [comment]
Related to #6863

File: pages/linux/dnstracer.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+`dnstracer -s {{dns.example.org}} {{www.example.com}}`
+
+- Only query IPv4 servers:
+
+`dnstracer -4 {{www.example.com}}`
+
+- Retry each request 5 times on failure:
+
+`dnstracer -r 5 {{www.example.com}}`
```
Comment (CleanMachine1) [inline pages/linux/dnstracer.md:20]:
```suggestion
`dnstracer -r {{5}} {{www.example.com}}`
```

**Review by CleanMachine1 — APPROVED** [review]
lgtm thanks

File: pages/linux/dnstracer.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# dnstracer
+
+> The dnstracer command determines where a DNS gets its information from.
+> More information: <http://www.mavetju.org/unix/dnstracer.php>.
```
Comment (reinhart1010) [inline pages/linux/dnstracer.md:4]:
```suggestion
> More information: <https://manned.org/dnstracer>.
```

Here, we prefer using the `dnstracer`'s official [`man`](https://manned.org/man) documentation as hosted in manned.org. See #5143 for more info.

**Review by reinhart1010 — CHANGES_REQUESTED** [review]
Thanks @Faro1991, I left some comments here for you to review.

File: pages/linux/dnstracer.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+- Find out where your local DNS got the information on www.example.com:
+
+`dnstracer {{www.example.com}}`
+
+- Start with a specific DNS that you already know:
```
Comment (kbdharun) [inline pages/linux/dnstracer.md:10]:
```suggestion
- Start with a [s]pecific DNS that you already know:
```

File: pages/linux/dnstracer.md (hunk 2 above)
Comment (Faro1991) [reply pages/linux/dnstracer.md:4]: Fine with me. I had linked the home page, because it was referred on the Kali tools page. `man` seems more sensible though.

File: pages/linux/dnstracer.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+`dnstracer -s {{dns.example.org}} {{www.example.com}}`
+
+- Only query IPv4 servers:
+
+`dnstracer -4 {{www.example.com}}`
+
+- Retry each request 5 times on failure:
```
Comment (kbdharun) [inline pages/linux/dnstracer.md:18]:
```suggestion
- [r]etry each request 5 times on failure:
```

File: pages/linux/dnstracer.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+`dnstracer -r {{5}} {{www.example.com}}`
+
+- Display all steps during execution:
+
+`dnstracer -v {{www.example.com}}`
+
+- Display an overview of all received answers after execution:
```
Comment (kbdharun) [inline pages/linux/dnstracer.md:26]:
```suggestion
- Display an [o]verview of all received answers after execution:
```

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution. I have some suggestions for this page.

File: pages/linux/dnstracer.md (hunk 5 above)
Comment (Faro1991) [reply pages/linux/dnstracer.md:26]: Understood, will keep in mind for future commits 🙂

File: pages/linux/dnstracer.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+- Start with a [s]pecific DNS that you already know:
+
+`dnstracer -s {{dns.example.org}} {{www.example.com}}`
+
+- Only query IPv4 servers:
+
+`dnstracer -4 {{www.example.com}}`
+
+- [r]etry each request 5 times on failure:
```
Comment (kbdharun) [inline pages/linux/dnstracer.md:18]: Hey, Sorry for the request can you revert this change?
```suggestion
- Retry each request 5 times on failure:
```
As `r` means `retries` your original version of the command description is fine as the mnemonic wouldn't be applicable here.

Comment (Faro1991) [reply pages/linux/dnstracer.md:18]: Of course, makes sense. Personally, I'd be fine with either, but it's of course preferable to be consistent with other pages.

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

> Also, Welcome to tldr btw.

**Comment by kbdharun** [comment]
Hey, @Faro1991 seems like you haven't signed the CLA. Can you sign the Contributor License Agreement (CLA) at <https://cla-assistant.io/tldr-pages/tldr>?

**Comment by Faro1991** [comment]
> Hey, @Faro1991 seems like you haven't signed the CLA. Can you sign the Contributor License Agreement (CLA) at <https://cla-assistant.io/tldr-pages/tldr>?

Hey, I did on my first PR, the check just took unusually long this time. 😅

**Comment by kbdharun** [comment]
> > Hey, @Faro1991 seems like you haven't signed the CLA. Can you sign the Contributor License Agreement (CLA) at https://cla-assistant.io/tldr-pages/tldr?
> 
> Hey, I did on my first PR, the check just took unusually long this time. 😅

Thanks, Now that all checks are clear. This PR is good to merge.

> Yeah, recently it is taking a lot of time maybe due to a heavy load this Hacktoberfest.

**Review by reinhart1010 — APPROVED** [review]
Thanks!
